.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /.idea
+Cargo.lock
```

Comment:
Apologies, I mean saving `Cargo.lock` in the repo is good but committing `Cargo.lock` in this PR isn't.
We are committing `Cargo.lock` when a dependencies update is needed like #266.

Please keep `.gitignore` and `Cargo.lock` untouched.